io/src/main/scala/sbt/io/IOSyntax.scala

```diff
@@ -0,0 +1,13 @@
+package sbt.io
+
+trait IOSyntax extends PathExtra /* with URIExtra */ {
+  type File = java.io.File
+  type URI = java.net.URI
+  type URL = java.net.URL
+
+  def uri(s: String): URI = new URI(s)
+  def file(s: String): File = new File(s)
+  def url(s: String): URL = new URL(s)
+}
+
+object syntax extends IOSyntax
```

Comment:
Trait allows us to bake `sbt.io.syntax` cake for immediate usage, while reserving the opportunity to bake `sbt` package object when all modules are ready.